fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -497,6 +484,34 @@ public void execute() throws Exception {
         }
     }
 
+    private void handleCreateTableAsSelectStmt(long beginTimeInNanoSecond) throws Exception {
+        CreateTableAsSelectStmt createTableAsSelectStmt = (CreateTableAsSelectStmt) parsedStmt;
+
+        // if create table failed should not drop table. because table may already exists,
+        // and for other cases the exception will throw and the rest of the code will not be executed.
+        createTableAsSelectStmt.createTable(context);
+        try {
+            com.starrocks.sql.analyzer.Analyzer analyzer =
```

Comment:
Did not see the analyze of CreateTableAsSelectStmt statement

Reply:
it is only support new Analyzer form CTASAnalyzer to deal it.